packages/logs-syslog/logs-syslog.dev~mirage/opam

```diff
@@ -15,11 +15,13 @@ depends: [
   "logs"
   "ptime"
   "syslog-message" {>= "0.0.2"}
-  "result"
```

Comment:
there is still a "result" in https://github.com/hannesm/logs-syslog/blob/master/opam